src/components/StrongPasswordInput/strengthBar.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import RenderIf from '../RenderIf';
+import { StyledStrenghtBar, StyledStateBar, StyledStateLabel } from './styled';
+
+export default function StrengthBar(props) {
+    const { passwordState, passwordStateLabel } = props;
+
+    let label;
+    if (passwordStateLabel) {
+        label = passwordStateLabel[passwordState];
+    }
```

Comment:
You should avoid making let statements every time you can.
We have to optimize everything we can, so use useMemo and the other hooks every time you can
Here you can do:

`const label = useMemo(
        () => (passwordStateLabel ? passwordStateLabel[passwordState] : undefined),
        [passwordState, passwordStateLabel],
    );`